drake/examples/Atlas/config/control_config_sim.yaml

```diff
@@ -0,0 +1,409 @@
+kinematic_tree_metadata:
+  body_names:
+    feet:
+      r: &R_FOOT_NAME r_foot
+      l: &L_FOOT_NAME l_foot
+    hands:
+      r: &R_HAND_NAME r_hand
+      l: &L_HAND_NAME l_hand
+
+  joint_group_names:
+    legs:
+      r: [r_leg_hpz, r_leg_hpx, r_leg_hpy, r_leg_kny, r_leg_aky, r_leg_akx]
+      l: [l_leg_hpz, l_leg_hpx, l_leg_hpy, l_leg_kny, l_leg_aky, l_leg_akx]
+    knees:
+      r: r_leg_kny
+      l: l_leg_kny
+    ankles:
+      r: [r_leg_aky, r_leg_akx]
+      l: [l_leg_aky, l_leg_akx]
+    arms:
+      r: [r_arm_shz, r_arm_shx, r_arm_ely, r_arm_elx, r_arm_uwy, r_arm_mwx, r_arm_lwy]
+      l: [l_arm_shz, l_arm_shx, l_arm_ely, l_arm_elx, l_arm_uwy, l_arm_mwx, l_arm_lwy]
+    neck: [neck_ay]
+    back_bkz: back_bkz
+    back_bky: back_bky
+    
+joint_names:
+  - back_bkz
+  - back_bky
+  - back_bkx
+  - neck_ay
+  - l_leg_hpz
+  - l_leg_hpx
+  - l_leg_hpy
+  - l_leg_kny
+  - l_leg_aky
+  - l_leg_akx
+  - r_leg_hpz
+  - r_leg_hpx
+  - r_leg_hpy
+  - r_leg_kny
+  - r_leg_aky
+  - r_leg_akx
+  - l_arm_shz
+  - l_arm_shx
+  - l_arm_ely
+  - l_arm_elx
+  - l_arm_uwy
+  - l_arm_mwx
+  - l_arm_lwy
+  - r_arm_shz
+  - r_arm_shx
+  - r_arm_ely
+  - r_arm_elx
+  - r_arm_uwy
+  - r_arm_mwx
+  - r_arm_lwy
+
+qp_controller_params:
```

Comment:
This is the replacement for the atlasParams and valkyrieParams Matlab files. It's a complete description of all of the parameters used in all of the operating modes, and there's an equivalent file which has the slightly different params we use for hardware. I'm abusing YAML slightly in order to do this: rather than writing out all of the params for every joint for every mode, I've specified defaults for everything, and then I'm only explicitly writing out the differences from the defaults. Since that might make it hard to read, the InstQP constructor automatically generates an expanded version of this file, called `control_config_sim.debug.yaml` which has every default expanded and the appropriate changes substituted in. That file should be easier to understand, but it's 2400 lines long for sim, and over 7000 lines long for the hardware params, so attempting to edit it directly or check it into the repo would be a bad idea.